Tools/MiniBrowserSwiftUI/Shared/BrowserApp.swift

```diff
@@ -28,6 +28,15 @@ import SwiftUI
 @main
 struct BrowserApp: App {
     var body: some Scene {
-        BrowserScene()
+        BrowserScene().commands {
+            CommandGroup(before: CommandGroupPlacement.newItem, addition: {
+                Button(action: {
+                    let currentWindow = NSApp.keyWindow
+                    currentWindow!.windowController!.newWindowForTab(nil)
+                    currentWindow!.addTabbedWindow(NSApp.keyWindow!, ordered: .above)
+                }, label: { Text("New Tab") })
+                .keyboardShortcut("t", modifiers: [.command])
```

Comment:
Command modifier is implied, this can just be
```suggestion
                .keyboardShortcut("t")
```